plugins/basic/nexus-timeline-plugin/src/main/java/org/sonatype/nexus/timeline/feeds/internal/DefaultFeedContentRenderer.java

```diff
@@ -54,7 +54,8 @@ public String getTitle(final FeedEvent evt) {
     if (title != null) {
       return title;
     }
-    return plainTextRenderer.getTitle(evt);
+    // TODO: Some human-readable fallback?
+    return evt.getEventType() + ":" + evt.getEventSubType();
```

Comment:
Is the logic in the plainTextRenderer no longer pertinent? We're still using it to format content so might be preferable to continue using it and deprecate logic there rather than C&P the simple version here